src/Common/Config/ConfigProcessor.h

```diff
@@ -92,9 +92,17 @@ class ConfigProcessor
         const zkutil::EventPtr & zk_changed_event,
         bool fallback_to_preprocessed = false);
 
+#if USE_SSL
```

Comment:
Could you move l. 95-102 further down to l. 116? Also, these methods can be made `static`, right?

Reply:
Done